main_test.go

```diff
@@ -53,6 +53,10 @@ func TestScripts(t *testing.T) {
 				env.Vars = append(env.Vars, fmt.Sprintf(`LOCALAPPDATA=%s\appdata`, env.WorkDir))
 			}
 
+			// In order to make go available inside containers where the guest
+			// and host OS and arch match, we define HOST_GOROOT
+			env.Vars = append(env.Vars, "HOST_GOROOT="+runtime.GOROOT())
```

Comment:
Perhaps I'm being a bit nitpicky here, but... this is the GOROOT at the time when dockexec was built, not the GOROOT that the current install uses.

What if someone downloads a dockexec release, or installs it via their package manager? There's no guarantee that the user will happen to have the same GOROOT path.

I think `go env GOROOT` is your best bet.

Reply:
This is in `main_test.go` so I think, per our chat, this is fine.